USAGE.md

```diff
@@ -127,7 +127,8 @@ Array
 )
 ```
 
-An Exception is thrown if an unsupported value is supplied for the $keyNames parameter.
+An `UnexpectedValueException` is thrown if an unsupported value is
+supplied for the `$keyNames` parameter.
```

Comment:
I don't think we need to be super strict in this, especially since there's no single correct way to break an English sentence into semantically (semi-)autonomous clauses. I'd be OK with us simply applying this principle in whatever changes in prose happen to be included in pull requests. 

That said, a separate commit applying the "standard" in this PR would be fine, I guess, since it's been discussed in this thread already (so it would be linked to relevant historical record). I can add one if you think it's worth it.

Reply:
Sure, please give it a shot.

Reply:
If it is better suited to #134, you could include the commit there instead.

Reply:
I'm actually thinking it might be worth a separate PR after all: we could introduce a `CONTRIBUTING.md` file, which is relatively standard in repos these days. That would allow us to, among other things, (1) explicitly mention the expectation of semantic line breaks, including a guideline of what should be considered a "long" line; and (2) streamline `GOVERNANCE.md` so it can be focused on maintainer-targeted guidelines, and leave contributor-targeted guidelines (such as [3, 4 and 5](https://github.com/hamstar/Wikimate/blob/master/GOVERNANCE.md#guidelines-for-commits-and-pull-requests)) next to the other contribution instructions.

So let's merge this one, and I'll prepare a new PR for `CONTRIBUTING.md` and include the line-breaking changes there.

Reply:
Update: PR submitted as #135 (which already links to this conversation).